src/nupic/os/Timer.cpp

```diff
@@ -66,38 +66,6 @@ static UInt64 getCurrentTime()
   return (UInt64)(v.QuadPart) - initialTicks_;
 }
 
-#elif defined(NTA_PLATFORM_darwin)
```

Comment:
This is important to figure out. For instance, what if the platform variables that control types got messed up and that is the cause of the bug?